libs/components/inline-form/src/lib/modules/inline-form/inline-form.component.ts

```diff
@@ -132,152 +137,130 @@ export class SkyInlineFormComponent implements OnInit, OnDestroy {
-                    styleType: 'link',
-                    action: 'cancel',
-                  },
-                ]);
-              }
-            );
-          });
+        ).subscribe((values: string[]) => {
+          emitter.next([
+            {
+              text: values[0],
+              styleType: 'primary',
+              action: 'done',
+            },
+            {
+              text: values[1],
+              styleType: 'link',
+              action: 'cancel',
+            },
+          ]);
+        });
         break;
 
       case SkyInlineFormButtonLayout.SaveCancel:
         observableZip(
           this.#resourcesService.getString('skyux_inline_form_button_save'),
           this.#resourcesService.getString('skyux_inline_form_button_cancel')
-        )
-          .pipe(take(1))
-          .subscribe((values: string[]) => {
-            promise = new Promise<SkyInlineFormButtonConfig[]>(
-              (resolve: (value: SkyInlineFormButtonConfig[]) => void) => {
-                resolve([
-                  {
-                    text: values[0],
-                    styleType: 'primary',
-                    action: 'save',
-                  },
-                  {
-                    text: values[1],
-                    styleType: 'link',
-                    action: 'cancel',
-                  },
-                ]);
-              }
-            );
-          });
+        ).subscribe((values: string[]) => {
+          emitter.next([
+            {
+              text: values[0],
+              styleType: 'primary',
+              action: 'save',
+            },
+            {
+              text: values[1],
+              styleType: 'link',
+              action: 'cancel',
+            },
+          ]);
+        });
         break;
 
       case SkyInlineFormButtonLayout.DoneDeleteCancel:
         observableZip(
           this.#resourcesService.getString('skyux_inline_form_button_done'),
           this.#resourcesService.getString('skyux_inline_form_button_delete'),
           this.#resourcesService.getString('skyux_inline_form_button_cancel')
-        )
-          .pipe(take(1))
-          .subscribe((values: string[]) => {
-            promise = new Promise<SkyInlineFormButtonConfig[]>(
-              (resolve: (value: SkyInlineFormButtonConfig[]) => void) => {
-                resolve([
-                  {
-                    text: values[0],
-                    styleType: 'primary',
-                    action: 'done',
-                  },
-                  {
-                    text: values[1],
-                    styleType: 'default',
-                    action: 'delete',
-                  },
-                  {
-                    text: values[2],
-                    styleType: 'link',
-                    action: 'cancel',
-                  },
-                ]);
-              }
-            );
-          });
+        ).subscribe((values: string[]) => {
```

Comment:
Does this subscription need a `takeUntil(this.#ngUnsubscribe)`?

Reply:
done in: [pipe takeUntil unsubscribe on resource string subscriptions](https://github.com/blackbaud/skyux/pull/670/commits/d720487ef67be7bf88df3aba4ae495d50b5ee070)